### DISCUSSION TITLE
Regolith 3 Wayland - touchpad settings not saved

### DISCUSSION THREAD

**Discussion by eljejer** [regolith-linux/regolith-desktop#905] [Q&A]
When I modify touchpad settings in Modify Settings (Super+c), these are not saved after a reboot. Log out and log back in, they are saved.

**Comment by JakobRieckh** [comment]
Same here. Touchpad settings are reset after each `regolith-look refresh`. Also on Regolith 3.0 Wayland.

**Reply by SoumyaRanjanPatnaik** [reply]
I just saw this. Can reproduce this issue. For me, the scroll settings persist, but the touchpad settings don't. I'll try to come up with a patch for this.

**Comment by SoumyaRanjanPatnaik** [comment]
Should be fixed in Regolith 3.1
